launch/launch/main.py

```diff
@@ -35,14 +35,38 @@ def main(argv=sys.argv[1:]):
         type=file_exists,
         nargs='+',
         help='The launch file.')
+    parser.add_argument(
+        '--args',
+        metavar="arg",
+        type=str,
+        nargs='+',
+        help='An argument to the launch file (e.g., arg_name:=value). All ' +
+        'arguments will be passed to each launch file.')
```

Comment:
I just re-ran the linter, and fixed associated errors as well as removing the chunk of useless code. Thanks for the reviews